common-ui/src/main/res/layout/fragment_components_dialogs.xml

```diff
@@ -71,6 +71,28 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"/>
 
+        <com.duckduckgo.mobile.android.ui.view.text.SectionHeaderTextView
+            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
+            android:text="Bottom Sheet Dialogs"
+            android:layout_marginTop="16dp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <com.duckduckgo.mobile.android.ui.view.button.ButtonSecondaryLarge
+            android:id="@+id/actionBottomSheetButton"
+            android:text="Action Bottom Sheet"
+            android:layout_marginTop="16dp"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
+
+        <com.duckduckgo.mobile.android.ui.view.button.ButtonSecondaryLarge
```

Comment:
can you add samples of PromoBottomSheet without image? basically we want to show all the different combinations for both components.